static/server/taskbar.css

```diff
@@ -37,14 +40,22 @@ li.ui-state-focus {
 	text-decoration: underline;
 }
 
-@media only screen and (min--moz-device-pixel-ratio: 1.5),
-only screen and (-o-min-device-pixel-ratio: 1.5/1),
-only screen and (-webkit-min-device-pixel-ratio: 1.5),
-only screen and (min-device-pixel-ratio: 1.5) {
-    #kiwixtoolbar button, #kiwixtoolbar input {
-    	font-size: 2em;
+/* the breakpoint "600px" is the width of the 3 btns "library", "home" and "randoml" */
+@media (max-width: 600px) {
+    .kiwixsearch {
+        float: none;
+        margin-top: 5px;
     }
+
     #kiwixsearchbox {
-		width: 7em;
-	}
+        width: calc(100% - 75px);
```

Comment:
`calc` is part of CSS3 values module and this module is still experimental.

We don't (and cannot) know what will be the browser version the users will have but `kiwix-serve` is mainly deployed where people has no access to the most recent hardware. 
So wee should avoid to use too complex or recent CSS functionality.
I think we can do almost the same things with a `float` + `padding` or `margin` attributes.